app.go

```diff
@@ -83,6 +83,12 @@ func main() {
 		Desc:   "Addresses to connect to the consumer queue (URLs).",
 		EnvVar: "DOCUMENT_STORE_API_ADDRESS",
 	})
+	apiHost := app.String(cli.StringOpt{
```

Comment:
I wonder if this string should be incorporated into other constants from the application. like for example: `authorityPrefix`, `methodeAuthority`, `ftBrand`.

Reply:
extracted.